main/activity-log.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+/** @typedef {import('./typings').ActivityEvent}  ActivityEvent */
+/** @typedef {import('./typings').ActivityEntry}  ActivityEntry */
```

Comment:
@juliangruber Would you like me to use this style of type-imports in all other places where I am referring to ActivityEvent and ActivityEntry types?

Reply:
Done in [4fd20c4](https://github.com/filecoin-project/filecoin-station/pull/110/commits/4fd20c419e7d3d584ae374e3c379aab4fce968fd).

I used typedefs only for the lines I am already changing to avoid mixing unrelated changes & extra code churn.